go/tasks/plugins/k8s/kfoperators/mpi/mpi.go

```diff
@@ -69,11 +71,14 @@ func (mpiOperatorResourceHandler) BuildResource(ctx context.Context, taskCtx plu
 	common.OverrideDefaultContainerName(taskCtx, podSpec, kubeflowv1.MPIJobDefaultContainerName)
 
 	// workersPodSpec is deepCopy of podSpec submitted by flyte
-	// WorkerPodSpec doesn't need any Argument & command. It will be trigger from launcher pod
 	workersPodSpec := podSpec.DeepCopy()
+	workerSpecCommand := []string{}
+	if val, ok := taskTemplateConfig["worker_spec_command"]; ok {
+		workerSpecCommand = strings.Split(val, " ")
+	}
 
 	for k := range workersPodSpec.Containers {
-		workersPodSpec.Containers[k].Args = []string{}
+		workersPodSpec.Containers[k].Args = workerSpecCommand
 		workersPodSpec.Containers[k].Command = []string{}
```

Comment:
Should we use `workerSpecCommand` for container's command? If not, I think we could rename it to `workerSpecArgs`
```suggestion
		workersPodSpec.Containers[k].Args = []string{}
		workersPodSpec.Containers[k].Command = workerSpecCommand
```

Reply:
I followed the same pattern as `_get_container` of `PythonAutoContainer`, where we put [get_command](https://github.com/flyteorg/flytekit/blob/master/flytekit/core/python_auto_container.py#L186) in `args` of the container. 

I think on the user-facing side (flytekit), we can call them commands, but on the backend side (flytepropeller), we can put the command in args.

Putting either in command or args of the container both work I guess. I just want to make it consistent with the other cases